src/listeners/messageCreate.ts

```diff
@@ -67,13 +67,44 @@ client.on("messageCreate", async message => {
 
 	// Delete non-stringURL messages in review-strings
 	const stringURLRegex = /(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi
+
 	if (message.channel instanceof TextChannel && message.channel.name.endsWith("-review-strings")) {
-		if (!/(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi.test(message.content))
-			await message.delete()
-		else {
-			await message.react("vote_yes:839262196797669427")
-			await message.react("vote_maybe:839262179416211477")
-			await message.react("vote_no:839262184882044931")
+		if (stringURLRegex.test(message.content)) {
+			const urlsWithComments: string[] = [],
+				contentSplit = message.content.split(" "),
+				areUrls = contentSplit.map(w => stringURLRegex.test(w))
+
+			let commentAtStart = "",
+				encounteredUrl = areUrls[0]
+
+			for (let i = 0; i < areUrls.length; i++) {
+				if (areUrls[i]) {
+					if (!encounteredUrl) {
+						urlsWithComments.push(`${commentAtStart} ${contentSplit[i]}`)
+						encounteredUrl = true
```

Comment:
How does this work? If you don’t find a url then the next one has to be an url?

Reply:
What do you mean by not find an url? You always find an url because of the if on line 72/

Reply:
That’s actually not what I meant but I just saw line 81 so nvm